continuedev/continue
# Add Pause Codebase On Start setting

## Description

Adds a Pause Codebase On Start VSCode user setting.

## Checklist


## Screenshots

<img width="784" alt="image" src="https://github.com/user-attachments/assets/7b3aeb6a-26d3-427c-81d0-ed9e1621d530">


## Testing

1. Set the Pause Codebase On Start setting to true
2. Open a Repository or hit Reload Window
3. The Codebase should not Index
4. Check that Manual Indexing still works
